### ISSUE TITLE
BBCode is sent via email

### ISSUE THREAD

**Issue by mokraemer** [mantisbt-plugins/BBCodePlus#109]
Mantis emails are always plain text.
The BBCode replaced in the mail are formated for HTML output, so you have html code inside a plain text mail.

**Comment by bkraul** [comment]
You are correct, by default, Mantis emails are set to plain text. However, there are manual ways to override this in the mantis installation (outside of the scope of BBCode+ plugin).

For this, the plugin does support attaching the code in the email but it also has an option to disable all of that from the emails. You can find this in the options:

![image](https://github.com/mantisbt-plugins/BBCodePlus/assets/14616851/75a47fc4-a2dc-4a8d-b679-39cef436967d)

**Comment by mokraemer** [comment]
do you have a link on how to make mantis send html mails?
I did not find anything here. Maybe some help in "Process email" on how to enable it for mantis would be nice for other users.

**Comment by bkraul** [comment]
As I said, enabling HTML email is outside of the scope of the plugin. It involves modifying/overriding the mantisbt source code. There have been a few approaches during the years. I am not sure if at this point there exists a plugin or maybe even a configuration inside of mantis itself. That would be a question for the maintainers.

You can start here:

https://www.mantisbt.org/bugs/view.php?id=1400

**Comment by bkraul** [comment]
Closed as not in-scope and because of age.
